Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -102,8 +127,24 @@ private void DispatchToValidationFailureRepository (IBusinessObjectValidationRes
 
     protected override bool EvaluateIsValid ()
     {
-      // TODO RM-6056: Shows a validation error if IBusinessObjectValidationResult returned messages for this control.
-      return true;
+      var sb = new StringBuilder();
+
+      foreach (var validationFailure in _localValidationFailures)
+      {
+        sb.Append("\r\n");
+        sb.Append(validationFailure.ErrorMessage);
+      }
+
+      if (_numberOfNotVisibleValidationErrors > 0)
+      {
+        var resourceString = ((IControlWithResourceManager)GetControlToValidate()).GetResourceManager().GetString(BocList.ResourceIdentifier.RemainingValidationFailureText);
```

Comment:
naming: `remainingValidationFailureText` to express the intention of the variable instead of just saying the "type" of string this is.